robosuite_env.py

```diff
@@ -466,7 +525,7 @@ def py3main():
         env.close()
 
     def py2main():
-        RealSawyerLift(grip_thresh=.1) #restore_rcan, rcan_kwargs, env_kwargs)
+        RealSawyerLift(grip_thresh=.1, use_image=args.image, use_proprio=args.proprio, port1=args.port1, joint_proprio=args.algo not in ['bc', 'lmp']) #restore_rcan, rcan_kwargs, env_kwargs)
```

Comment:
Same comment as before, if these args dont really affect anything, lets simplify what we have to run in the command line